scripts/gui/microbe_editor.mjs

```diff
@@ -229,38 +238,37 @@ function setRedo(enabled){
 }
 
 //! Sends organelle selection to the Game
-function onSelectNewOrganelle(organelle){
+function onSelectNewOrganelle(element){
 
-    if(common.isInEngine()){
+    if(common.isInEngine() && !$(element.element).hasClass("DisabledButton")) {
```

Comment:
This should be in the upper function. This is could for example be called from the editor, so this *must* take the organelle name as a parameter.